src/network/networkd-dhcp6.c

```diff
@@ -10,6 +10,7 @@
 
 #include "sd-dhcp6-client.h"
 
+#include "conf-parser.h"
```

Comment:
Not a matter, but it is not necessary to include this here, as networkd-dhcp6.h already includes it.